monitor/grafana/zeebe.json

```diff
@@ -373,7 +373,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "1 - kube_pod_container_status_ready{pod=~\".*\", pod!~\".*curator.*|worker.*|starter.*\", namespace=\"$namespace\"}",
+              "expr": "1 - kube_pod_container_status_ready{pod=~\"$pod\", pod!~\".*curator.*|worker.*|starter.*\", namespace=\"$namespace\"}",
```

Comment:
Does it mean that elastic is no longer showed here?

Reply:
Is still available, but cannot be filtered on as it is not one of the pods.
<img width="1283" alt="Screen Shot 2022-01-24 at 10 49 00" src="https://user-images.githubusercontent.com/3511026/150759849-d76a4a56-fa4e-411e-9c19-59e62c67cf96.png">